spec/index.bs

```diff
@@ -2954,9 +3010,14 @@ The {{GPUMapMode}} flags determine how a {{GPUBuffer}} is mapped when calling
                 [=Content timeline=] steps:
 
                 1. Let <var data-timeline=content>contentTimeline</var> be the current [=Content timeline=].
+                1. If |this|.{{GPUBuffer/mapState}} is not {{GPUBufferMapState/"unmapped"}}:
+                    1. Return [=a promise rejected with=] an {{OperationError}}.
```

Comment:
Hm, I thought this was a non-change from the existing spec but it's slightly different because this rejects immediately instead of later.